src/Standards/Generic/Docs/Commenting/DocCommentStandard.xml

```diff
@@ -0,0 +1,394 @@
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: One blank line between the short and long descriptions.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *<em>
+ * Long description.
+ */
+        ]]>
+        </code>
+        <code title="Invalid: More than one blank line between the short and long descriptions.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *
+ *
+</em>
+ * Long description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The long description must start with a capital letter.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Long description starts with a capital letter.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * <em>L</em>ong description.
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Long description doesn't start with a capital letter.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * <em>l</em>ong description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There must be one blank line before the tags.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: One blank line before the tags.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *</em>
+ * @param int $foo
+ */
+        ]]>
+        </code>
+        <code title="Invalid: More than one or no blank line before the tags.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *
+ *</em>
+ * @param int $foo
+ */
+
+ /**
+ * Short description.</em></em>
+ * @return int
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Parameter tags must be grouped together.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Parameter tags grouped together.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+<em> * @param int $foo
+ * @param string $bar</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Parameter tags not grouped together.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * @param int $foo
+<em> *</em>
+ * @param string $bar
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Parameter tags must not be grouped together with other tags.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Parameter tags are not grouped together with other tags.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * @param int $foo
+ *
+<em> * @author Some Author
+ * @return int</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Parameter tags grouped together with other tags.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+<em> * @param  int $foo
+ * @author Some Author
+ * @return int</em>
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There must be a single blank line after a tag group.
```

Comment:
```suggestion
    Tag groups must be separated by a single blank line.
```

Reason for this subtle change: there should be no blank line at the end of a DocBlock (i.e. after the last tag group).

Reply:
@jrfnl, I just realized that the error message is the same as what I had used in the documentation. Should I update it as well, either in this PR or in a separate PR?

The error code is called `SpacingAfterTagGroup`, but I guess that is harder to change as it would be a breaking change.

https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/84acf4e56f110db8e75cb9a575c5727df637643c/src/Standards/Generic/Sniffs/Commenting/DocCommentSniff.php#L293

Reply:
In my opinion, the error message in the sniff is fine as it is, as it won't be thrown when the tag group is the last content in the docblock, so won't cause confusion. That's different for the docs.